### PR TITLE
EVE-1156 fix issue with pip install on worker boot

### PR DESCRIPTION
**Pull request by tcarmet**

### CODE REVIEW

**Comment by bertranddemiddelaer** [comment]
> # Not authorized
> I'm afraid I cannot do that, @bertranddemiddelaer:
> 
> > /bypass_build_status
> 
> You don't have enough credentials to use `bypass_build_status`.
> 
> Please **edit** or **delete** the corresponding comment so I can move on.

Oh well, nevermind, I am repo admin.

**Comment by bertranddemiddelaer** [comment]
> # Incorrect fix version
> The `Fix Version/s` in issue EVE-1156 contains:
> 
> * _None_
> 
> Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:
> 
> * `1.7.9`
> 
> Please check the `Fix Version/s` of EVE-1156, or the target
> branch of this pull request.

fixed

File: eve/bin/os_script_init.sh [hunk 1]
```diff
@@ -54,9 +52,15 @@ then
     # buildbot > 0.9.12 requires a version of twisted that is not supported
     # by python2.6
     worker_version=0.9.12
+    python_version=2.6
   else
     twisted_version=16.4.0
+    python_version=2.7
   fi
+
+  curl https://bootstrap.pypa.io/pip/${python_version}/get-pip.py -o get-pip.py
```
Comment (ghost) [inline eve/bin/os_script_init.sh:63]: What is the reason to not use 'pip install --reinstall pip==9.0.3' ?
Do we really want to rely on another externally hosted script for thoat ?
Thats worth a comment at least to me

**Pull request merged**